lib/api/include/irods/replica_truncate.h

```diff
@@ -0,0 +1,87 @@
+#ifndef IRODS_REPLICA_TRUNCATE_H
+#define IRODS_REPLICA_TRUNCATE_H
+
```

Comment:
I think we need the `/// \file` directive?

Reply:
Done